cmd/bootnode/README.md

```diff
@@ -0,0 +1,23 @@
+Using custom bootnode
+======================
+
+To make custom bootnode usable it must participate in the same DHT as the target cluster.
+
+1. Generate private key for a node
+
+To generate a private key you can use bootnode command line tool from go-ethereum repository.
```

Comment:
It may still require some searching. I would include this snippet:
```
docker run --entrypoint=bootnode -v $(pwd):/keys ethereum/client-go:alltools-latest -genkey /keys/node.key
```